packages/manager/src/components/VerticalLinearStepper/VerticalLinearStepper.styles.ts

```diff
@@ -0,0 +1,53 @@
+import { StepConnector, StepIcon } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { isPropValid } from 'src/utilities/isPropValid';
+
+type StyledCircleIconProps = {
+  activeStep: number;
+  index: number;
+};
+
+export const StyledCircleIcon = styled('div', {
+  label: 'StyledCircleIcon',
+  shouldForwardProp: (prop) => isPropValid(['backgroundColor', 'border'], prop),
```

Comment:
You don't seem to be passing in `backgroundColor` and `border` as props into this styled component when you're using it in `VerticalLinearStepper.tsx` -- `activeStep` and `index` are being passed instead 

```suggestion
  shouldForwardProp: (prop) => isPropValid(['activeStep', 'index'], prop),
```